its/src/test/java/com/sonar/it/csharp/UCFGDeserializationTest.java

```diff
@@ -63,39 +63,41 @@ public static void initializeOrchestrator() throws Exception {
     OrchestratorBuilder builder = Orchestrator.builderEnv()
       .setSonarVersion(Optional.ofNullable(System.getProperty("sonar.runtimeVersion")).filter(v -> !"LTS".equals(v)).orElse("LATEST_RELEASE[6.7]"));
 
-    String securityVersion = System.getProperty("csharpVersion");
+    String csharpVersion = System.getProperty("csharpVersion");
     Location csharpLocation;
-    if (StringUtils.isEmpty(securityVersion)) {
+    if (StringUtils.isEmpty(csharpVersion)) {
       // use the plugin that was built on local machine
       csharpLocation = FileLocation.byWildcardMavenFilename(new File("../sonar-csharp-plugin/target"), "sonar-csharp-plugin-*.jar");
     } else {
       // QA environment downloads the plugin built by the CI job
-      csharpLocation = MavenLocation.of("org.sonarsource.dotnet", "sonar-csharp-plugin", securityVersion);
+      csharpLocation = MavenLocation.of("org.sonarsource.dotnet", "sonar-csharp-plugin", csharpVersion);
     }
-    builder.addPlugin(csharpLocation);
+    builder
+      .addPlugin(csharpLocation)
+      .addPlugin(MavenLocation.of("com.sonarsource.security", "sonar-security-plugin", "1.0.0.847"))
+      .addPlugin(MavenLocation.of("com.sonarsource.license", "sonar-dev-license-plugin", "3.3.0.1341"));
```

Comment:
These two plugin versions are hard-coded but the csharp one is parameterised. Can these be parameterised too?

Reply:
I have no idea. I doubt we need to parametrize the license plugin version - it is for private use and there does not seem to be a need to test against multiple versions.

For the security plugin, perhaps, I don't know. I suggest updating that if there is a specific need.

It seems the parameter for the sonar csharp plugin was added to enable the QA job to run against the version that was built by the CI job... We don't run ITs against multiple versions of the C# plugin.